.github/workflows/code-quality.yml

```diff
@@ -55,6 +55,11 @@ jobs:
       - name: Run Lint Check
         run: pnpm lint
 
+      # TODO: uncomment this once type issues are fixed
```

Comment:
locally, the `check-types` script that i added has no errors. however, in CI, there are quite a few: https://github.com/recallnet/js-recall/actions/runs/18735220676/job/53440391325#step:10:23

not sure what settings we need in the turbo file etc, but i _have_ noticed that `apps/comps` doesn't properly emit warnings for things like `any`. created a follow up [ENG-751](https://linear.app/recall-labs/issue/ENG-751/fixup-codebase-types-viacheck-types-script-and-re-enable-script-in-ci).

Reply:
For sure something is wrong in CI because all the lint errors in that log are just incorrect. All those variables do have types when things are pnpm installed, compiled, etc correctly.

Apart from that, I agree, I've also noticed more lax treatment of `any` in the comps app, but that is a separate issue I think.

Reply:
Oh, quick guess, but maybe you need to add `check-types` to the root turbo.json and make sure it has `^build` as a dependency.